### DISCUSSION TITLE
How to find rotated image.

### DISCUSSION THREAD

**Discussion by Dineshkumar-Anandan-ZS0367** [PaddlePaddle/PaddleOCR#12449] [Q&A]
How can you identified rotated image for extracting text using this ocr.

Which code can i refer in this github

I need rotated angle and how to rotate as proper image

**Comment by GreatV** [comment]
for Text Direction Classification, you can refer: [Text Direction Classification.](https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/angle_class_en.md)
